x-pack/plugins/apm/server/routes/services/annotations/index.ts

```diff
@@ -41,6 +48,16 @@ export async function getServiceAnnotations({
     searchAggregatedTransactions,
     start,
     end,
+  }).catch((error) => {
+    if (
+      error instanceof WrappedElasticsearchClientError &&
+      error.originalError instanceof errors.RequestAbortedError
+    ) {
+      return [];
+    }
```

Comment:
I don't think we need to special-case this. Errors from aborted requests are handled at the route level.